src/OrbitQt/ConnectToStadiaWidget.cpp

```diff
@@ -5,6 +5,7 @@
 #include "ConnectToStadiaWidget.h"
 
 #include <absl/strings/str_format.h>
+#include <grpc/impl/codegen/connectivity_state.h>
```

Comment:
Is clangd suggesting that? Looks like an internal header to me. I assume iwyu suggests something else instead.

Reply:
Yes it was clangd. iwyu did not complain when it was in there. I removed it and everything still works, also iwyu is not complaining as far as I can tell